exist-core/src/main/resources/org/exist/xquery/lib/xqsuite/xqsuite.xql

```diff
@@ -958,7 +958,7 @@ declare %private function test:assertXPath($annotation as element(annotation), $
                 function ($namespaces as map(*), $xml as element()) {
                     map:merge(($namespaces,
                 	    for $prefix in in-scope-prefixes($xml)
-                	    where $prefix != "" and $prefix != "xml"
```

Comment:
Interestingly this existing code probably didn't do what the author intended anyway (due to a common misunderstanding of the `!=` operator). If you want to see what I mean, just run:
```
let $prefix := ""
return
  $prefix != ""
,
let $prefix := "xml"
return
  $prefix != "xml"
```